prep-javascript-objects/main.js

```diff
@@ -0,0 +1,17 @@
+var person = {
+  firstName: 'Meet',
+  lastName: 'Patel',
+  hobby: 'Coding'
+};
+
+console.log(person);
+console.log(`The Person's name is: ${person.firstName}`);
```

Comment:
Try to combine the person's first and last name into one variable with the `+` operator.